skbio/sequence/_sequence.py

```diff
@@ -887,7 +933,22 @@ def __str__(self):
         return str(self._string.decode("ascii"))
 
     def __repr__(self):
-        """Return a string representation of the biological sequence object.
+        r"""Return a string representation of the biological sequence object.
```

Comment:
Right, they'll never see it in the built HTML docs. They'll see it by calling `help` or `?` in an interactive python session or by reading the code though :P